x-pack/legacy/plugins/uptime/public/lib/helper/stringify_kueries.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
+ * Extract a map's keys to an array, then map those keys to a string per key.
+ * The strings contain all of the values chosen for the given field (which is also the key value).
+ * Reduce the list of query strings to a singular string, with AND operators between.
+ */
+export const stringifyKueries = (kueries: Map<string, Array<number | string>>): string =>
+  Array.from(kueries.keys())
```

Comment:
The code we copy from other apps, which is being used in multiple apps, is there a common folder between plugins, where we can port code, i think, in that way, it will be easy to maintain.

Reply:
There is a shared `observability` plugin that we will be moving to in the future. The plan at this point is to get a working Kuery Bar into Uptime (the last Obs. plugin without it), and then work to extract them from all the plugins at once. The other bars are custom implementations as well. Basically once this project is concluded I'm going to move forward to try and get that ball rolling.